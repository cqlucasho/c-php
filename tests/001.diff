001+ hello_word_test_abc
001- ant extension is available
002+ helloWordIndexAbc
003+ HelloWordIndexAbd