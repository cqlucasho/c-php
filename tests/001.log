
---- EXPECTED OUTPUT
ant extension is available
---- ACTUAL OUTPUT
hello_word_test_abc
helloWordIndexAbc
HelloWordIndexAbd
---- FAILED
