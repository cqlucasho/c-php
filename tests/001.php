<?php
echo ant_camel("HelloWordTestAbc"),"\n";
//echo ant_pascal("hello_word", 0),"\n";
//echo ant_pascal("hello_word", 1),"\n";
echo ant_pascal("hello_word_index_abc", 0),"\n";
echo ant_pascal("hello_word_index_abd", 1),"\n";
?>
