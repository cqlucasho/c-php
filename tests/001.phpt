--TEST--
Check for ant presence
--SKIPIF--
<?php if (!extension_loaded("ant")) print "skip"; ?>
--FILE--
<?php
echo ant_camel("HelloWordTestAbc"),"\n";
//echo ant_pascal("hello_word", 0),"\n";
//echo ant_pascal("hello_word", 1),"\n";
echo ant_pascal("hello_word_index_abc", 0),"\n";
echo ant_pascal("hello_word_index_abd", 1),"\n";
?>
--EXPECT--
ant extension is available
