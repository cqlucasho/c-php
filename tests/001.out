hello_word_test_abc
helloWordIndexAbc
HelloWordIndexAbd